### PR TITLE
Wangjunwei blog fix some bugs

### PR DESCRIPTION
**Pull request by gavinwang23**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
title: "'Score Your Face Photo' a ML&Wechaty practice"
author: huyingxi
image: /assets/2020/your-blog-title/teaser.jpg
---

YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST

```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.js.org#contribute-guideline)

### CODE REVIEW

**Comment by gavinwang23** [comment]
Sorry,I found something went wrong in my blog,so I altered some files.please check and Merging it

File: _posts/2020-07-28-wechaty-puppet-douyin.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ categories: project
 image: /assets/2020/wechaty-puppet-douyin/wechaty-puppet-douyin-logo.jpg
```
Comment (lijiarui) [inline _posts/2020-07-28-wechaty-puppet-douyin.md]: Where is this file?

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: _posts/2020-07-28-wechaty-puppet-douyin.md (hunk 1 above)
Comment (gavinwang23) [reply _posts/2020-07-28-wechaty-puppet-douyin.md]: It's already in master
image

Comment (gavinwang23) [reply _posts/2020-07-28-wechaty-puppet-douyin.md]: Sorry, I didn’t check the article well before

File: _posts/2020-08-20-wechaty-puppet-douyin-mid-term.md [hunk 2]
```diff
@@ -58,7 +60,7 @@ tags:
   - 探索puppet更多的功能，实现底层autojs
   
   PPT:
-  <iframe src="https://youtu.be/zDIPzGztn_E" width="560" height="315" frameborder="0" allowfullscreen="" style=" position: absolute; top:0; left:0; width:100%; height:100%; "></iframe>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/zDIPzGztn_E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (huan) [inline _posts/2020-08-20-wechaty-puppet-douyin-mid-term.md]: I'd like to suggest that you can use the `iframe` new feature created from @univerone this week:

```html
{% include iframe.html src="https://www.youtube.com/embed/zDIPzGztn_E" %}
```

**Comment by lijiarui** [comment]
Please resolve the conflict first.

File: _posts/2020-08-20-wechaty-puppet-douyin-mid-term.md (hunk 2 above)
Comment (gavinwang23) [reply _posts/2020-08-20-wechaty-puppet-douyin-mid-term.md]: I saw that you have modified my blog, thank you very much, I have just modified the blog as your request

**Comment by gavinwang23** [comment]
I think everything is done😄

**Comment by gavinwang23** [comment]
Thanks for your help！

**Comment by lijiarui** [comment]
You should resolve the conflict first and then I can look into your blog.

**Pull request closed without merging**